docs/useAuthState.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: default
+title: "useAuthState"
+---
+
+# `useAuthState`
+
+To avoid rendering a component, and to force waiting for the `authProvider` response, use `useAuthState()` instead of `useAuthenticated()`. It calls `authProvider.checkAuth()` on mount and returns an object with 3 properties:
```

Comment:
```suggestion
To avoid rendering a component, and to force waiting for the `authProvider` response, use `useAuthState()` instead of `useAuthenticated()`. It calls `authProvider.checkAuth()` on mount and returns an object with 2 properties:
```